cmd/tier.go

```diff
@@ -61,6 +63,25 @@ func (config *TierConfigMgr) isTierNameInUse(tierName string) (madmin.TierType,
 	return madmin.Unsupported, false
 }
 
+func tierBackendInUse(sc madmin.TierConfig) (bool, error) {
+	var d warmBackend
+	var err error
+	switch sc.Type {
+	case madmin.S3:
+		d, err = newWarmBackendS3(*sc.S3)
+	case madmin.Azure:
+		d, err = newWarmBackendAzure(*sc.Azure)
+	case madmin.GCS:
+		d, err = newWarmBackendGCS(*sc.GCS)
+	default:
+		return false, errors.New("Unsupported tier type")
+	}
+	if err != nil {
+		return false, err
+	}
+	return d.InUse(context.Background())
```

Comment:
In a subsequent PR we should thread the context from adminAPIHandler down to warm backend in this case.